### DISCUSSION TITLE
How to set to undisplay the warning msg "Select content with Ctrl+left mouse button"

### DISCUSSION THREAD

**Discussion by wj-qiao** [sumatrapdfreader/sumatrapdf#4151] [General]
As shown in the title, how to undisplay the msg "Select content with Ctrl+left mouse button". And my sumatrapdf version is v3.5.2 64-bit.

Thank you.

**Comment by kjk** [comment]
I've made this auto-disappear after 2 secs in https://github.com/sumatrapdfreader/sumatrapdf/commit/52f80393d6d58ff9afc847d0d2935e2d4e691fb2

Does that help?

Any reason it bothers you so much? The intent is to help people discover functionality.

**Comment by wj-qiao** [comment]
The msg is of little use to me and appears constantly, which hinders document readability. It would be better if I could customize the display time. Changing it has indeed improved the situation significantly.

Thank you.
